packages/api-generator/src/utils.ts

```diff
@@ -133,15 +133,15 @@ async function getSources (name: string, locale: string, sources: string[]) {
 
   return {
     find: (section: string, key: string, ogSource = name) => {
+      const githubUrl = `https://github.com/vuetifyjs/vuetify/tree/${currentBranch}/packages/api-generator/src/locale/${locale}/${ogSource}.json`
```

Comment:
This is broken for things that dont have a normal local ogSource 
eg: useDisplay ogSource is `@vue/reactivity/dist/reactivity.d.ts#L451-L459` instead of `useDisplay` or VDataTable slots that point to sources like `vuetify/src/labs/VDataTable/VDataTableHeaders.tsx#L38-L47`